app/internal/agenda_events.py

```diff
@@ -1,28 +1,35 @@
 from datetime import date, timedelta
 from typing import List, Optional
 
-from app.database.models import Event
-from app.database.database import SessionLocal
 import arrow
 from sqlalchemy.exc import SQLAlchemyError
+from sqlalchemy.orm import Session
+
+from app.database.models import Event
+from app.internal.utils import get_all_user_events
 
 
 def get_events_per_dates(
-        session: SessionLocal,
+        session: Session,
         user_id: int,
         start: Optional[date],
```

Comment:
why is it optional? In the event table, it was not nullable.

Reply:
It's not my code, but i think it's optional because it is a GET request, so when the user views the page for the first time he has not selected any event to view. therfor "start" and "end" needs to be optional